samples/get_view_image.py

```diff
@@ -0,0 +1,67 @@
+####
+# This script demonstrates how to use the Tableau Server Client
+# to query a high resolution image of a view from Tableau Server.
+#
+# For more information, refer to the documentations on 'Query View Image'
+# (https://onlinehelp.tableau.com/current/api/rest_api/en-us/help.htm)
+#
+# To run the script, you must have installed Python 2.7.X or 3.3 and later.
+####
+
+import argparse
+import getpass
+import logging
+
+import tableauserverclient as TSC
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description='Query View Image From Server')
+    parser.add_argument('--server', '-s', required=True, help='server address')
```

Comment:
@t8y8 i think you are saying "don't assume that everything is in the default site." yes?